### ISSUE TITLE
Why pathPrefix does not work for gifs in pages's md after deploy

### ISSUE THREAD

**Issue by FengShangWuQi** [gatsbyjs/gatsby#4148]
I add a pathPrefix with project name instead of /, but I found that can not find gif files correctly after deploy using github pages, it returns 404, however, pictures in pages's md returns 200.

network looks like this:
png: ...github.io/pathPrefix/static/path/xx.png -> 200
gif: ...github.io/xx.gif -> 404

**Comment by KyleAMathews** [comment]
Hmmm looks like https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-remark-copy-linked-files/src/index.js needs updated to support path prefixes. Is this something you'd be interested in fixing? Would love to help you make a PR!

**Comment by FengShangWuQi** [comment]
Thanks a lot :thumbsup:, waiting for your good news, and I also find that the files in static/ in the same problem after add pathPrefix.

**Comment by FengShangWuQi** [comment]
@KyleAMathews Is this problem solved?

**Comment by KyleAMathews** [comment]
@FengShangWuQi no one has written a PR for it so no. Would love it if you wanted to look into solving the issue! See our contributing guide https://www.gatsbyjs.org/docs/how-to-contribute/

**Comment by ajrussellaudio** [comment]
I've also hit this issue. Can I suggest that this issue is given a more descriptive name to reflect that is doesn't only affect GIF images, but all files with relative links inside Markdown after a deploy with pathPrefix.

I'm going to have a crack at a PR to fix this, but it's my first time digging into the Gatsby internals so I won't have instant results.

**Comment by ajrussellaudio** [comment]
...actually this has been fixed. If anyone is still searching for this , update your `package.json` to require a newer version of `gatsby-remark-copy-linked-files`, version 1.5.31 worked for me.

```json
{
  "dependencies": {
    "gatsby-remark-copy-linked-files": "^1.5.31",
  }
}
```

**Comment by pieh** [comment]
as @ajrussellaudio mentioned - this was fixed with #4841
